config/owasp/suppressions.xml

```diff
@@ -173,6 +173,10 @@
         <packageUrl regex="true">^pkg:npm/flat@.*$</packageUrl>
         <cve>1085318</cve>
     </suppress>
+    <suppress>
```

Comment:
May be there's a resolution for it i.e. adding its version 5.0.2 in package.json and then reinstall yarn?

Reply:
fixed